src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -10,7 +10,7 @@ namespace Internal.Runtime.CompilerHelpers
     /// <summary>
     /// These methods are used to throw exceptions from generated code.
     /// </summary>
-    internal static class InteropHelpers
+    public static class InteropHelpers
```

Comment:
Basically the same reason we modified types to be public in Net Native multi-file. We root public surface area.  When compiling as library code these methods are not included in compilation so I flipped the type to be public.  Alternatively we could add declarative roots for them in the compiler (yuck) or try to compile all types marked internal and above (not confident it'll succeed generally).